### PR TITLE
v7.x

### PR DESCRIPTION
**Pull request by carlosescri**
Related to issue https://github.com/doofinder/support/issues/70

Second part of the PR #94 
Actual PR will mix actual changes in BK-685 into master, going with all the adjustments done in PR #94 plus:

- Travis deletion
- New Tag creation
- `composer.json` update version

@carlosescri as you created this PR i cant place you as a reviewer, but please, if you can check too if everything that will be merged looks fine for you... Thanks! :smiley:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,158 +0,0 @@
-# Changelog
```
Comment (CAxGu) [inline CHANGELOG.md:1]: Its ok to delete this? or should i restore it before merge?

File: swagger/conf.json [hunk 2]
```diff
@@ -1 +0,0 @@
-{"invokerPackage": "DoofinderManagement", "packagePath": "Management"}
```
Comment (CAxGu) [inline swagger/conf.json:1]: All swagger stuff is missing, would we maintain only the composer build?

**Review by JoeZ99 — APPROVED** [review]

File: swagger/conf.json (hunk 2 above)
Comment (CAxGu) [reply swagger/conf.json:1]: Reviewed this in meet with @carlosescri ,we will it

File: CHANGELOG.md (hunk 1 above)
Comment (CAxGu) [reply CHANGELOG.md:1]: Reviewed this in meet with @carlosescri ,we will restore this file before merge and include last changes that we have done

File: composer.json [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
+  "version": "7.0.0",
   "type": "library",
   "description": "Doofinder PHP API Client",
   "keywords": [
     "search",
     "api",
     "doofinder"
   ],
-  "homepage": "https://github.com/doofinder/php-doofinder",
+  "homepage": "https://github.com/doofinder/php-doofinder-2.0",
```
[the remark this answers is not in the document]
Comment (CAxGu) [reply composer.json:11]: Reviewed this in meet with @carlosescri ,we will mantain master value

**Pull request merged**